plutus-core/executables/src/PlutusCore/Executable/Parsers.hs

```diff
@@ -4,10 +4,16 @@
 
 module PlutusCore.Executable.Parsers where
 
-import PlutusCore.Executable.Common
+import PlutusCore.Executable.Types
 
 import Options.Applicative
 
+-- | Invert a switch: return False if the switch is supplied on the command
+-- line, True otherwise.  This is used for command-line options to turn things
+-- off.
+invertedSwitch :: Mod FlagFields Bool -> Parser Bool
+invertedSwitch = fmap not . switch
```

Comment:
alternative implementation: `flag False True`

Reply:
Ah, maybe that would be more compact.  I'll try that.